CMakeLists.txt

```diff
@@ -340,6 +340,7 @@ if(WITH_REENTRANT_STRSIGNAL)
     set(HAVE_REENTRANT_STRSIGNAL 1 CACHE INTERNAL "Reentrant strsignal is supported.")
 endif(WITH_REENTRANT_STRSIGNAL)
 
+option(WITH_ROCKSDB "Use system rocksdb" OFF)
```

Comment:
might want to `s/WITH_ROCKSDB /WITH_DLIBROCKSDB/` ?

and shall fail the configure phase if librocksdb is not found?